pylearn2/linear/tests/test_cudnn.py

```diff
@@ -0,0 +1,151 @@
+import theano
+from theano import tensor
+from theano.sandbox import cuda
+
+from pylearn2.linear.conv2d import Conv2D, make_random_conv2D
+from pylearn2.linear.cudnn2d import Cudnn2D
+from pylearn2.space import Conv2DSpace
+from pylearn2.utils import sharedX
+from pylearn2.testing.skip import skip_if_no_gpu
+
+import unittest
+from nose.plugins.skip import SkipTest
+
+import numpy as np
+
+skip_if_no_gpu()
+
+if cuda.cuda_available is False:
+    raise SkipTest('Optional package cuda disabled')
```

Comment:
Instead of these last 4 lines, it would be better to call `skip_if_no_gpu()` inside `TestCudnn.setUp`.
This should not interfere with the script checking the docstring format.